graylog2-web-interface/src/views/logic/TimeRange.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import type { TimeRange, RelativeTimeRange } from 'views/logic/queries/Query';
```

Comment:
Have you considered writing tests for this file? Maybe it is enough to test this functionality by testing the logic that implements the functions provided by this file. But I still see a benefit in writing unit tests, specifically for this file.

Reply:
:white_check_mark: